app/routers/symbol_settings.py

```
@@ -0,0 +1,500 @@
+        buy_prices=buy_prices,
+        quantity_per_order=float(settings_obj.buy_quantity_per_order),
+        currency=currency,
+    )
+
+    return EstimatedCostResponse(**result)
+
+
+@router.get("/symbols/all/estimated-cost", response_model=AllEstimatedCostResponse)
+async def get_all_estimated_costs(
+    request: Request,
+    db: AsyncSession = Depends(get_db),
```

Ruff (0.14.6)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)